### ISSUE TITLE
front end for next 24 hours

### ISSUE THREAD

**Issue by AdelaideBaron** [AdelaideBaron/condition-checker#2]
Create a front end to display the information from the condition checker for the next 24 hours

- to be for a phone screen initially, not dynamic... 
- Will <eventually> have a calendar for people to select from 
- show the current date & time 

Once the above is done, we will: (make a new ticket)
- extract the id to display an icon on the page 
- add a calendar for people to select dates 
- display the ConditionResponse in a raw format for now 
- This will have the date & time as null, and therefore just call the generic endpoint /get_conditions
